extensions/apps/notifications/src/index.tsx

```diff
@@ -5,28 +5,26 @@ import {
   IntegrationRegistrationOptions,
   LogoTypeSource,
   IRootComponentProps,
+  IPlugin,
 } from '@akashaorg/typings/lib/ui';
 import { NotificationPlugin } from './plugins/notification-plugin';
-import getSDK from '@akashaorg/awf-sdk';
-import { filter, from, map, mergeMap } from 'rxjs';
-import { AUTH_EVENTS } from '@akashaorg/typings/lib/sdk';
 import { Logger } from '@akashaorg/awf-sdk';
 import React from 'react';
 import { BellIcon } from '@akashaorg/design-system-core/lib/components/Icon/hero-icons-outline';
 
 export const initialize = (options: IntegrationRegistrationOptions & { logger: Logger }) => {
   const notification: any = options.plugins['@akashaorg/app-notifications']?.notification;
-  const sdk = getSDK();
+  // const sdk = getSDK();
 
   if (notification && typeof notification.notify === 'function') {
     notification.listenLogin(() => {
-      const markAsRead$ = sdk.api.globalChannel.pipe(
-        filter(
-          data =>
-            data.event === AUTH_EVENTS.MARK_MSG_READ ||
-            data.event === AUTH_EVENTS.NEW_NOTIFICATIONS,
-        ),
-      );
+      // const markAsRead$ = sdk.api.globalChannel.pipe(
```

Comment:
i have noticed such commented out lines accumulate and become stale over time, i suggest to remove them